app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,19 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int
+                 ) -> None:

```

Comment:
```

    def __init__(
             self,
             nickname: str,
             favourite_dish: str,
             hummer_level: int
     ) -> None:
```

Reply:
There is an error in your proposed solution Flake8 "Closing bracket does not match visual indentation"

Reply:
There should be no error. The closing bracket should be on the same level as `def` - then everything will be fine